wallet/src/Pos/Wallet/Web/Pending/Worker.hs

```diff
@@ -99,11 +99,12 @@ resubmitPtxsDuringSlot sendActions ptxs = do
 processPtxsToResubmit
     :: MonadPendings m
     => SendActions m -> SlotId -> [PendingTx] -> m ()
-processPtxsToResubmit sendActions curSlot ptxs = do
+processPtxsToResubmit sendActions _curSlot ptxs = do
     ptxsPerSlotLimit <- evalPtxsPerSlotLimit
     let toResubmit =
-            take ptxsPerSlotLimit $
-            filter ((curSlot >=) . view ptxNextSubmitSlot) $
```

Comment:
Keep an eye on this removal. If we're doing everything with the status ie `PtxApplying` vs `PtxWontApply` then the `ptxNextSubmitSlot` may not matter.